pygeo/parameterization/BaseDVGeo.py

```diff
@@ -0,0 +1,218 @@
+"""
+BaseDVGeo
+
+Holds a basic version of a DVGeo geometry object
+Enables the use of different geometry parameterizations (FFD, OpenVSP, ESP, etc) with the MACH-Aero framework
+"""
+
+from abc import abstractmethod
+from typing import OrderedDict
+
+
+class BaseDVGeometry:
```

Comment:
Does this need to be a ABC to work?

Reply:
It should be `ABC`, but it doesn't have to be for it to work, that just prevents this class from being instantiated directly. The overloaded methods will still work without `ABC`.

Reply:
But you make A an `ABC` then you get this error
```
Traceback (most recent call last):
  File "test.py", line 23, in <module>
    myA = A()
TypeError: Can't instantiate abstract class A with abstract methods test
```
Which seems more useful to me

Reply:
I see what you mean now, yes you are right. We should be using `ABC` here.

Reply:
Oh hold on I didn't catch this, I don't think `DVGeoMulti` should inherit anything since it encapsulates DVGeo objects.

Reply:
DVGeoMulti has been set back to its original state before this PR.